### PR TITLE
Added eslint command to package.json and fixed linting issues

### PR DESCRIPTION
**Pull request by Onomanatee**
https://jira.itpservices.be/browse/DESSSYS-40

### CODE REVIEW

File: packages/create-itp-react-app/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "prompt": "^1.0.0",
     "replace-in-file": "^3.4.2",
     "shelljs": "^0.8.2",
-    "recursive-copy": "^2.0.9"
+    "recursive-copy": "^2.0.9",
```
Comment (brecht) [inline packages/create-itp-react-app/package.json]: invalid json

File: packages/create-itp-react-app/templates/app/.eslintrc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "extends": "./node_modules/@inthepocket/itp-react-scripts/dist/config/eslintrc.js",
+    "parser": "babel-eslint",
```
Comment (thibmaek) [inline packages/create-itp-react-app/templates/app/.eslintrc]: why do we need to switch the parser here?

File: packages/create-itp-react-app/templates/app/config-overrides.js [hunk 3]
```diff
@@ -3,17 +3,17 @@ const rewirePostCss = require('react-app-rewire-postcss');
 
 module.exports = {
   webpack: (config, env) => {
-    config = rewireCssModules.webpack(config, env, {
+    let newConfig = rewireCssModules.webpack(config, env, {
```
Comment (thibmaek) [inline packages/create-itp-react-app/templates/app/config-overrides.js]: 👍

File: packages/create-itp-react-app/templates/app/config-overrides.js [hunk 4]
```diff
@@ -3,17 +3,17 @@ const rewirePostCss = require('react-app-rewire-postcss');
...
       test: /\.module.css$/
     });
 
-    config = rewirePostCss(config, {
-      plugins: loader => [
-        require('postcss-preset-env')(),
+    newConfig = rewirePostCss(newConfig, {
+      plugins: loader => [ // eslint-disable-line no-unused-vars
```
Comment (thibmaek) [inline packages/create-itp-react-app/templates/app/config-overrides.js]: why not just `() => []` instead?

File: packages/create-itp-react-app/templates/app/.eslintrc (hunk 2 above)
Comment (Onomanatee) [reply packages/create-itp-react-app/templates/app/.eslintrc]: The generated catalog code needs it to get through the linter.

More specifically, these errors:
```
.../.../design-docs/catalog/components/itp-rcc-autocomplete/index.js
  12:9  error  Parsing error: Unexpected token =

.../.../design-docs/catalog/index.js
  4:19  error  Parse errors in imported module './pages': Unexpected token import (18:49)  import/no-named-as-default
  4:19  error  Parse errors in imported module './pages': Unexpected token import (18:49)  import/no-named-as-default-member

.../.../design-docs/catalog/pages.js
  18:49  error  Parsing error: Unexpected token import
```

File: packages/create-itp-react-app/templates/app/config-overrides.js (hunk 4 above)
Comment (Onomanatee) [reply packages/create-itp-react-app/templates/app/config-overrides.js]: Eh, I kinda like the verbosity here, makes it clear what the code is about. I can remove it though. :)

File: packages/create-itp-react-app/templates/app/.eslintrc (hunk 2 above)
Comment (thibmaek) [reply packages/create-itp-react-app/templates/app/.eslintrc]: OK!

File: packages/create-itp-react-app/templates/app/config-overrides.js (hunk 4 above)
Comment (thibmaek) [reply packages/create-itp-react-app/templates/app/config-overrides.js]: fine by me

**Pull request merged**